roles/container-engine/cri-o/vars/fedora.yml

```diff
@@ -5,4 +5,4 @@ crio_packages:
 
 crio_service: cri-o
 crio_conmon: /usr/libexec/crio/conmon
-crio_seccomp_profile: ""
```

Comment:
why removed? then `default` will kick in  with `crio_seccomp_profile: "/etc/crio/seccomp.json"`
But for fedora there is no seccomp_profile file (some dists delivers a modified form), so keep it empty

Reply:
Yes, this has nothing to do with this PR, removed it in latest commit.